code/modules/research/designs/boards/machine_engie.dm

```diff
@@ -14,6 +14,16 @@
 	category = "Machine Boards"
 	build_path = /obj/item/weapon/circuitboard/smes
 
+/datum/design/treadmill
```

Comment:
Yes, it's a generator, still doesn't mean it's the best location, since engineering won't use it, for the R-UST and P.A.C.M.A.N., engineering DOES use those (arguably)